torch/_inductor/compile_fx.py

```diff
@@ -1083,9 +1086,9 @@ def cudagraphify_impl(
     """
     Assumes inputs[static_input_idxs[i]] are always the same memory address
     """
-    check_input_idxs = get_input_idxs_to_check(inputs, static_input_idxs)
-    static_input_idxs = remove_unaligned_input_idxs(inputs, static_input_idxs)
-    copy_misaligned_inputs(inputs, check_input_idxs)
+    check_input_idxs = get_input_idxs_to_check(inputs, static_input_idxs)  # type: ignore[arg-type]
```

Comment:
`get_input_idxs_to_check` taking Sequence would fix this..

Reply:
Right, if `get_input_idxs_to_check` took in a Sequence it would be covariant. anyway, not really important, just commenting